visual-inspection/inspection-frontend/src/components/InspectImage.vue

```diff
@@ -60,8 +63,14 @@ export default {
 
 <i18n>
 {
-  "de": { "answer": "„Das ist {object}“"},
-  "en": { "answer": "\"This is {object}\""}
+  "de": {
+     "answer1": "„Das ist zu {percentage}",
+     "answer2": " % {object}“"
```

Comment:
Wir brauchen hier glaube ich zwei alternative Strings: Einmal einen mit Wahrscheinlichkeit/Konfidenz und einmal einen ohne. Es sollte auch nicht nötig sein, das in zwei Hälften zu trennen, sondern es können zwei Variablen in einem einzelnen String sein.

Reply:
okay, mach ich. 
ich hab das eigentlich aufgeteilt, weil das bei mir mit den zwei Variablen aus irgendeinem Grund nicht geklappt hat. Aber ich schaus mir nochmal an. 

Dann mach ich auch gleich das mit der Konfiguration, welches angezeigt wird. 

Und die Tests schau ich mir dann auch gleich an. 
Danke!